config/routes.rb

```diff
@@ -1,8 +1,12 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'posts/index'
+  get 'posts/show'
+  get 'users/index'
+  get 'users/show'
 
```

Comment:
- [ ] Do we really need these routes when we have defined them below? To keep your code clean, kindly remove these unnecessary routes.